### ISSUE TITLE
Always show option to delete own reply

### ISSUE THREAD

**Issue by traumschule** [Joystream/pioneer#4031]
For some reason own (empty) forum reply can't be removed.

This causes that empty posts due to #3967 cannot be cleaned up by the author but have to be removed by the forum moderator.

**Comment by traumschule** [comment]
To access the context menu the conditions `isOwn && isActive` had to be removed [here](https://github.com/Joystream/pioneer/blob/dev/packages/ui/src/forum/components/PostList/PostContextMenu.tsx#L64):
`return <ContextMenu title="Post actions" size="small" items={getContextMenuItems()} />`

Still deleting fails:
```
Transaction error: Withdraw, ExtrinsicFailed
on network: wss://rpc.joystream.org
in block: 0x557eeec01d56bc77ea511db47a78ed91d678e3b3312bcf0c9e54f6086444ba51
more details at: https://polkadot.js.org/apps/?rpc=wss://rpc.joystream.org#/explorer/query/0x557eeec01d56bc77ea511db47a78ed91d678e3b3312bcf0c9e54f6086444ba51
```
> forum.PostDoesNotExist
> Post does not exist.

![tx-failed](https://user-images.githubusercontent.com/31551045/211932418-76fbf558-4047-41fe-afad-22d90f467390.png)

![delete-fail](https://user-images.githubusercontent.com/31551045/211931852-8e1dd4b7-9074-4919-be77-9987893ca993.png)

Questions:
To delete a post does it have to be editable? That would explain why it doesn't appear in the according runtime map.
Should the forum lead be able to moderate [this post](https://pioneerapp.xyz/#/forum/thread/7?post=33) nonetheless: `{categoryId:36,threadId:7,postId:33}`

**Comment by dmtrjsg** [comment]
Most likely If the post is non-editable it should not be possible to delete it... @Lezek123 can you pls kindly confirm?

**Comment by Lezek123** [comment]
Yes, if it's a non-editable post then the author cannot remove it currently. It can only be removed by the forum moderator through `workerRemark` using metaprotocol (as explained here: https://discord.com/channels/811216481340751934/815192060276506664/1063194158849863791)
